packages/allure-js-commons/src/sdk/utils.ts

```diff
@@ -56,13 +63,31 @@ export const getMessageAndTraceFromError = (error: Error | { message?: string; s
   };
 };
 
-export const allureIdRegexp = /(?:^|\s)@?allure\.id[:=](?<id>[^\s]+)/;
-export const allureIdRegexpGlobal = new RegExp(allureIdRegexp, "g");
+type AllureTitleMetadataMatch = RegExpMatchArray & {
+  groups: {
+    type?: string;
+    v1?: string;
+    v2?: string;
+    v3?: string;
+    v4?: string;
+  };
+};
+
+export const allureTitleMetadataRegexp = /@?allure\.(?<type>\S+)[:=]("[^"]+"|'[^']+'|`[^`]+`|\S+)/;
```

Comment:
We should bind this RE to avoid polynomial complexity in a way similar to how we did it for id and label REs (see [here](https://github.com/allure-framework/allure-js/pull/1065)).

This time we can't fully prevent backtracking, but it will be limited to three occurrences (one per each type of quote), which is OK.

```suggestion
export const allureTitleMetadataRegexp = /(?:^|\s)@?allure\.(?<type>\S+)[:=]("[^"]+"|'[^']+'|`[^`]+`|\S+)/;
```

Reply:
I think, it wouldn't be unreasonable 👍